conda_smithy/configure_feedstock.py

```diff
@@ -254,14 +254,14 @@ def apply_migrations(list_of_metas, root_path):
     """
     migrations_root = os.path.join(root_path, 'migrations', '*.yaml')
     migrations = glob.glob(migrations_root)
-    
+
     from .variant_algebra import parse_variant, variant_add
-    
+
     migration_variants = [(fn, parse_variant(open(fn, 'r').read())) for fn in migrations]
-    migration_variants.sort(key = lambda fn_v: (fn_v[1].get('migration_ts'), fn_v[0]))
+    migration_variants.sort(key = lambda fn_v: (-fn_v[1]['migration_ts'], fn_v[0]))
```

Comment:
Pretty sure we do want to sort by oldest first so probably not a minus sign here.